doc/api/v8.md

```diff
@@ -1,17 +1,29 @@
 # V8
 
-    Stability: 2 - Stable
+The `v8` module exposes APIs that are specific to the version of [V8][]
+built into the Node.js binary. It can be accessed using:
 
-This module exposes events and interfaces specific to the version of [V8][]
-built with Node.js.  These interfaces are subject to change by upstream and are
-therefore not covered under the stability index.
+```js
+const v8 = require('v8');
+```
+
+The APIs are subject to change at any time.
```

Comment:
Ack
On May 18, 2016 1:59 AM, "Ben Noordhuis" <notifications@github.com> wrote:

> In doc/api/v8.md
> <https://github.com/nodejs/node/pull/6829#discussion_r63668151>:
>
> >
> > -This module exposes events and interfaces specific to the version of [V8][]
> > -built with Node.js.  These interfaces are subject to change by upstream and are
> > -therefore not covered under the stability index.
> > +```js
> > +const v8 = require('v8');
> > +```
> > +
> > +The APIs are subject to change at any time.
>
> Not just the APIs, their behavior as well. The paragraph you replaced
> captures it better, IMO.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/6829/files/cb4e88bdec4ad3af22e7a4e8d76e214a24eb56f8#r63668151>
>